src/components/EventCard.tsx

```diff
@@ -29,7 +29,7 @@ const DateText = styled(Text)`
 `;
 
 interface EventsCardProps {
-  src: string;
+  src?: string;
```

Comment:
Consider renaming this to `imageSrc` to be more descriptive, and consistent with naming conventions in the Announcement component.

Reply:
agree